pkg/sensors/accelerator/gpu/variables_habana.go

```diff
@@ -0,0 +1,33 @@
+//go:build gpu && habana
```

Comment:
I think if we use only the habana flag then we can drop nvml, dummy and dcgm sources from the `acceleratorOrder` variable file...

```
var (
	acceleratorOrder = []acceleratorInterface{
		&gpu_source.GPUHabana{},
	}
)
```